packages/api-client/README.md

```diff
@@ -57,7 +57,7 @@ client
     .catch(error => console.error(error));
 ```
 
-[CodeSandbox sample](https://codesandbox.io/s/api-clientget-started-be0xu?file=/src/index.js)
+[CodeSandbox sample](https://codesandbox.io/s/api-client-get-started-v2-1-4jpvd9)
```

Comment:
In the sand box, we try to access the key `results` in the promise resolution. But we should access `records` instead.